native/abnativeelf.cpp

```diff
@@ -38,15 +38,6 @@ constexpr std::array<uint8_t, 8> ar_thin_magic = {0x21, 0x3C, 0x74, 0x68,
 // {'B', 'C', '\xC0', '\xDE'}
 constexpr std::array<uint8_t, 4> llvm_bc_magic = {0x42, 0x43, 0xC0, 0xDE};
 
-// Loongson2F (MIPS-III) (= 0x20000000)
-constexpr uint32_t elf_flags_loongson2f = EF_MIPS_ARCH_3;
-
-// Loongson3 (MIPS) (= 0x80a20000)
-constexpr uint32_t elf_flags_loongson3 = EF_MIPS_ARCH_64R2 | EF_MIPS_MACH_GS464;
-
-// MIPS64R6EL (= 0xa0000400)
-constexpr uint32_t elf_flags_mips64r6el = EF_MIPS_ARCH_64R6 | EF_MIPS_NAN2008;
```

Comment:
We still want to use these for more robust flag tests.

You can compare the mask against these, if there is no match, an investigation is needed to see if `binutils` has changed behaviors